Hotel/Model/HotelManager.cs

```diff
@@ -15,17 +16,16 @@ public class HotelManager
         public RepositoryBackedObservableCollection<Room> Rooms { get; }
         public RepositoryBackedObservableCollection<Booking> Bookings { get; }
 
-        public HotelManager(IRepository<Guest> guestRepository, IRepository<Room> roomRepository, IRepository<Booking> bookingRepository)
+        public HotelManager([Dependency]IRepository<Guest> guestRepository, [Dependency]IRepository<Room> roomRepository, [Dependency]IRepository<Booking> bookingRepository)
```

Comment:
Is de Hotelmanager nog steeds noodzakelijk nadat we dependency injection hebben geimplementeerd?